.github/workflows/pr_ci.yaml

```diff
@@ -1,21 +1,31 @@
 name: pr_ci
 on:
   pull_request:
-    branches: [ "main" ]
-    
+    branches:
+      - main
+    types: [opened, reopened, synchronize]
+
+permissions:
+  contents: read
+
 jobs:
-  build:
+  ci:
+    name: Run PR check
     runs-on: ubuntu-latest
-
     steps:
       - uses: actions/checkout@v4
-      - name: Setup Go
-        uses: actions/setup-go@v5
+      - uses: actions/setup-go@v5
         with:
           go-version-file: 'go.mod'
 
       - name: Install dependencies
         run: go get .
-        
+
+      - name: Lint with golangci-lint
+        uses: golangci/golangci-lint-action@v6
+        with:
+          version: v1.60
+          only-new-issues: true
```

Comment:
Made the changes as per https://github.com/scribe-org/Scribe-Server/pull/14#issuecomment-2322995453 - though looking at this again.. decided to actually consolidate the initial linting changes into the existing workflow, `pr_ci.yaml`.

---

The additional modification to note here is this [`only-new-issues`](https://github.com/golangci/golangci-lint-action?tab=readme-ov-file#only-new-issues) option, which is to configure `golangci-lint-action` to only act on the new changes of a PR.